amundsen_application/static/js/ducks/utilMethods.ts

```diff
@@ -40,7 +40,7 @@ export function logClick(
   const target = event.currentTarget;
   const inferredProps: ActionLogParams = {
     command: 'click',
-    target_id: target.id,
+    target_id: target.dataset ? target.dataset.type : target.id,
```

Comment:
Oh also we must check `(target.dataset && target.dataset.type)`, I just tested a case that uses `id` and `target.dataset` is an empty object, so this got set to `undefined`.

Reply:
OK.

Where is it that we are still using ids? It might be better to just change those.

Reply:
It would be but I didn't want this PR to increase in scope, especially for changes in logging. I think that's more appropriate for a dedicated commit that gets explicitly shepherded to production and verified. Any issues or bugs risk us just losing that user event data for however long the issues existed in production.

Reply:
Comment may have gotten lost in all the comments here, but would like to file a ticket for this to use `dataset.type` instead of `id` for logging. I myself forget all the places without going through and looking for each of them but if you search for places where we call `logClick`, every component that triggers the event where we call `logClick` is where we'd make this change.

Reply:
OK, I will file a ticket!